guide/sharding/extended.md

```diff
@@ -26,32 +26,32 @@ client.on('message', message => {
 This will never work for a channel that lies on another shard. So, let's remedy this.
 
 ::: tip
-In discord.js v12, <DocsLink path="class/ShardClientUtil?scrollTo=ids">`client.shard`</DocsLink> can hold multiple ids. If you use the default sharding manager, the `.ids` array will only have one entry.
+In discord.js v13, <DocsLink path="class/ShardClientUtil?scrollTo=ids">`client.shard`</DocsLink> can hold multiple ids. If you use the default sharding manager, the `.ids` array will only have one entry.
 :::
 
 ```js {4-13}
 if (command === 'send') {
 	if (!args.length) return message.reply('please specify a destination channel id.');
 
-	return client.shard.broadcastEval(`
-		const channel = this.channels.cache.get('${args[0]}');
+	return client.shard.broadcastEval(async (client, { channelId }) => {
+		const channel = client.channels.cache.get(channelId);
 		if (channel) {
```

Comment:
This if statement should be written as:
```js
if (channel) {
	// ...
	return true;
}
return false;
```
Keeping the `channel.send()` line of course